packages/db/src/pages.ts

```diff
@@ -313,6 +314,12 @@ export async function findMessageResultPage(
 		}),
 	]);
 
+	const allMessageIds = messages.map((m) => m.id);
+
+	const attachments = await db.query.dbAttachments.findMany({
```

Comment:
Let's keep attachments on the message, not separate - if it's a bug let's fix the bug

Reply:
Fixed the bug! ^ 😄